app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int = 1,
+                 clean_mark: int = 1,
+                 brand: str = "") -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float = 1.0,
+                 clean_power: int = 1,
+                 average_rating: float = 1.0,
+                 count_of_ratings: int = 0):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
```

Comment:
these values have specific range of correct values. Let's check if they are correct, if not -> set some default correct value